dp_wizard/utils/code_generators/analyses/count/__init__.py

```diff
@@ -11,9 +14,12 @@
 has_bins = False
 
 
+root = Path(__file__).parent / "no-tests"
```

Comment:
`root `is defined in multiple places - could it be defined once and shared in the code?

Reply:
Maybe. I'll need to check, but these are resolving to separate directories. (There are multiple `no-tests` directories.) Maybe this whole convention is confusing? Let me take another look...

Reply:
Made a utility function, so hopefully it's a little less ugly?